src/js/components/Select/Select.js

```diff
@@ -264,7 +265,6 @@ const Select = forwardRef(
                   value={inputValue}
                   size={size}
                   theme={theme}
-                  onClick={disabled === true ? undefined : onRequestOpen}
```

Comment:
Why was this line removed?

Reply:
Because this event was already being called here:
https://github.com/grommet/grommet/blob/07203885d5da564695390038b09c8ddb9d81044e/src/js/components/Select/Select.js#L206

That's why the `onOpen` was being called twice.